lib/hijack/meteorx.js

```diff
@@ -0,0 +1,100 @@
+// Various tricks for accessing "private" Meteor APIs borrowed from the
+// now-unmaintained meteorhacks:meteorx package.
```

Comment:
If you're reviewing this module, you should compare the tricks implemented here to their original implemenations in https://github.com/meteorhacks/meteorx/tree/master/lib.